src/status_im/contexts/wallet/send/select_collectible_amount/view.cljs

```diff
@@ -45,9 +48,9 @@
        :status          (if incorrect-value? :error :default)
        :container-style style/network-tags-container}]
      [quo/amount-input
-      {:max-value       (controlled-input/upper-limit value)
-       :min-value       (controlled-input/lower-limit value)
-       :value           (controlled-input/numeric-value value)
+      {:max-value       (-> value controlled-input/upper-limit money/to-number)
+       :min-value       (-> value controlled-input/lower-limit money/to-number)
+       :value           (-> value controlled-input/numeric-value money/to-number)
```

Comment:
I have fixed this part in https://github.com/status-im/status-mobile/pull/21209. These methods are used in multiple places (swaps, collectibles).